modules/cnf-fast-event-notifications-api-refererence.adoc

```diff
@@ -63,6 +63,22 @@ Creates a new subscription. If a subscription is successfully created, or if it
 }
 ----
 
+[discrete]
+=== HTTP method
+
+`DELETE api/ocloudNotifications/v1/subscriptions`
+
+[discrete]
+=== Description
+
+Deletes all subscriptions.
+
+.Example API response
+[source,json]
+----
+TBD
```

Comment:
update

Reply:
Hi @jzding 
These updates don't belong here. They have been added by mistake. I have reverted these updates as they are being tracked in a separate PR. This PR is focused only on T-GM metrics.
Could you please take a look?
Thanks!